goldrush_path/goldrush_path.cpp

```diff
@@ -34,6 +35,27 @@
 #include <tuple>
 #include <vector>
 
+double sum_phred(const std::string& qual)
+{
+  double phred_sum = 0;
+  for (const auto& phred : qual) {
+    int phred_score = (int)(phred - 33);
+
+    // Delog the Phred score: 10^(-Q/10)
+    double delog_phred = pow(10.0, -phred_score / 10.0);
+
+    phred_sum += delog_phred;
+  }
+  return phred_sum;
+}
+
+uint32_t calc_median(std::vector<uint32_t>& vec, const size_t n)
+{
+  // sort vector in descending order
+  std::sort(vec.begin(), vec.end(), std::greater<uint32_t>());
+  return vec[n / 2];

```

Comment:
Might just be misunderstanding, but I thought that you said that no matter what the median is, the min that this threshold can be is 10? I'm not sure where that logic is?

Reply:
I think during the iteration, I must have removed it and forgot to put it back. Thanks for flagging

Reply:
done